src/daft-session/src/python.rs

```diff
@@ -1,13 +1,16 @@
+use std::sync::Arc;
+
 use daft_catalog::{
     python::{PyCatalogWrapper, PyIdentifier, PyTable, PyTableSource, PyTableWrapper},
     Identifier,
 };
+use daft_dsl::functions::python::WrappedUDFClass;
 use pyo3::prelude::*;
 
 use crate::Session;
 
 #[pyclass]
-pub struct PySession(Session);
+pub struct PySession(pub Session);
```

Comment:
I feel strongly about keeping the inner Session private. Do we have any alternatives rather than exposing this?

Reply:
well we need to get the inner session, so we can either make this public, or add

```rs
impl PySession {
    pub fn session(&self) -> &Session {
        &self.0
    }
}
```

Reply:
I chose to make it public as that's what we've done in most of the other `Py_` wrappers when we need to access the inner type. 

```rs
pub struct PyLogicalPlanBuilder {.
    pub builder: LogicalPlanBuilder,
}

pub struct PyMicroPartition {
    pub inner: Arc<MicroPartition>,
}

pub struct PyRecordBatch {
    pub record_batch: RecordBatch,
}

pub struct PyScanTask(pub Arc<ScanTask>);

// ... and so on

Reply:
Could you please use the helper method here. I understand other places might do this, but I would rather not expose private members unless absolutely necessary.